src/query/sql/src/executor/physical_plan_builder.rs

```diff
@@ -1605,7 +1605,7 @@ impl PhysicalPlanBuilder {
                     .reduce(|lhs, rhs| {
                         ScalarExpr::FunctionCall(FunctionCall {
                             span: None,
-                            func_name: "and".to_string(),
+                            func_name: "and_filters".to_string(),
```

Comment:
What's the different between the two funcs?

Reply:
`and_filters` will judge the mask if the error happens,  `number > 1 and  3 /(number - 1)` will not throw div by zero error because `number - 1` will not be zero in the first condition.